build_tools/torchscript_e2e_heavydep_tests/README.md

```diff
@@ -13,7 +13,7 @@ self-contained virtual environment. It can be used like so:
 # serialized test artifacts in the other specified directory.
 # This command is safe to re-run if you have already built the virtual
 # environment and just changed the tests.
-build_tools/e2e_heavydep_tests/generate_serialized_tests.sh \
+build_tools/torchscript_e2e_heavydep_tests/generate_serialized_tests.sh \
```

Comment:
The `torchscript_e2e_heavydep_tests` directory should be renamed to `e2e_heavydep_tests`. This seems to have been missed in https://github.com/llvm/torch-mlir/commit/e16b43e20b770ebbdfb536ff539bec8847537b59